bindata/nova/005-compute.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: nova.openstack.org/v1
+kind: NovaCompute
+metadata:
+  name: nova-compute-{{ .WorkerOspRole }}
+  namespace: openstack
+spec:
+  novaComputeCpuDedicatedSet: {{ .NovaComputeCPUDedicatedSet }}
+  novaComputeCpuSharedSet: {{ .NovaComputeCPUSharedSet }}
+  novaComputeImage: docker.io/tripleotrain/rhel-binary-nova-compute:current-tripleo
+  roleName: {{ .WorkerOspRole }}
+  serviceAccount: nova-operator
+  cinderPassword: TOBEREMOVED
+  internalAPIVip: TOBEREMOVED
+  memcacheServers: TOBEREMOVED
+  neutronPassword: TOBEREMOVED
+  novaPassword: TOBEREMOVED
+  placementPassword: TOBEREMOVED
+  publicVip: TOBEREMOVED
+  rabbitTransportURL: TOBEREMOVED
```

Comment:
removed with https://github.com/openstack-k8s-operators/nova-operator/pull/31 but we need 

commonConfigMap: common-config
ospSecrets: osp-secrets

which we can default to common-config/osp-secrets for now